src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java

```diff
@@ -100,7 +100,6 @@ private RetrievableStateHandle<JobGraph> getStateHandle(JobID jobId) throws Flin
 		}
     }
 
-	@Override
```

Comment:
It is was not simply removing this annotation, you have to properly implement `localCleanupAsync` and `globalCleanupAsync`